.github/workflows/run-benchmarks.yml

```diff
@@ -41,6 +41,10 @@ jobs:
       OUTPUT_NAME: benchmark-${{ matrix.browser }}-${{ matrix.buildType }}-${{ matrix.pageType }}.json
       INFURA_PROJECT_ID: ${{ secrets.INFURA_PROJECT_ID }}
       SENTRY_DSN_PERFORMANCE: ${{ vars.SENTRY_DSN_PERFORMANCE }}
+      # Environment variables for Sentry metrics reporting
+      BUILD_TYPE: ${{ matrix.buildType }}
```

Comment:
This is defined and then unused, since `matrix.buildType` is used later, which is fine

Reply:
removed now. good spot !